specification/maps/resource-manager/Microsoft.Maps/preview/2020-04-01-preview/maps-management.json

```diff
@@ -466,6 +466,253 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlasCreateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Private Atlas was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "201": {
+            "description": "The Private Atlas was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "202": {
+            "description": "The Private Atlas was accepted for creation",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates the Private Atlas resource. Only a subset of the parameters may be updated after creation, such as Tags.",
+        "operationId": "PrivateAtlas_Update",
+        "x-ms-examples": {
+          "UpdateAccount": {
+            "$ref": "./examples/UpdatePrivateAtlas.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateAtlasNameParameter"
+          },
+          {
+            "name": "PrivateAtlasUpdateParameters",
+            "in": "body",
+            "description": "The updated parameters for the Private Atlas.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlasUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Private Atlas was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "202": {
+            "description": "The Private Atlas was accepted for update",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "404": {
+            "description": "The specified Private Atlas was not found. Nothing was updated."
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "delete": {
+        "description": "Delete a Maps Account.",
```

Comment:
I bet you meant: "Delete a Private Atlas resource."

Reply:
clean up on most recent iteration